apps/meteor/app/lib/server/functions/loadMessageHistory.ts

```diff
@@ -29,7 +30,9 @@ export async function loadMessageHistory({
 		throw new Error('error-invalid-room');
 	}
 
-	const hiddenMessageTypes = getHiddenSystemMessages(room);
+	const hiddenSystemMessages = (await getSettingCached('Hide_System_Messages')) as MessageTypesValues[];
```

Comment:
Use the normal settings.get

On almost anything that's not used by microservices we can use settings.get (from CachedSettings)